Packs/CreateEDLInstance/.pack_ignore

```diff
@@ -0,0 +1,2 @@
+[file:README.md]
+ignore=RM106
```

Comment:
```suggestion
```
no need to keep it, the error is handled, on the other side it fails the validation

Reply:
removed